### DISCUSSION TITLE
Support ADDITIONAL_TRUSTED_DOMAIN in TALK

### DISCUSSION THREAD

**Discussion by mnaiman** [nextcloud/all-in-one#5147] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 
Using 2 domains with nextcloud-aio using env ADDITIONAL_TRUSTED_DOMAIN
apache container in config support redirect request to second domain in Caddy config

Talk container in /conf/signaling.conf support only one domain but clearly support more.

Please add env to talk container ADDITIONAL_TRUSTED_DOMAIN and propagate to config.

Thanks

**Comment by szaimen** [comment, marked as answer]
Hi, currently we don't have plans to add this as ADDITIONAL_TRUSTED_DOMAIN is mostly intended for internal usage with Nextcloud and not the talk container. However PRs are welcome!

**Reply by mnaiman** [reply]
PR is very easy. Update for file https://github.com/nextcloud/all-in-one/blob/main/Containers/talk/start.sh

[backend]
backends = backend-1,backend-2
allowall = false
timeout = 10
connectionsperhost = 8

[backend-1]
url = https://${NC_DOMAIN}
secret = ${SIGNALING_SECRET}
maxstreambitrate = ${TALK_MAX_STREAM_BITRATE}
maxscreenbitrate = ${TALK_MAX_SCREEN_BITRATE}

[backend-2]
url = https://${ADDITIONAL_TRUSTED_DOMAIN}
secret = ${SIGNALING_SECRET}
maxstreambitrate = ${TALK_MAX_STREAM_BITRATE}
maxscreenbitrate = ${TALK_MAX_SCREEN_BITRATE}

Can this be included?

**Reply by szaimen** [reply]
Can you create a PR?
